app/src/main/java/ai/elimu/analytics/EventListActivity.kt

```diff
@@ -21,6 +24,18 @@ class EventListActivity : AppCompatActivity() {
         binding = ActivityEventListBinding.inflate(layoutInflater)
         setContentView(binding.root)
 
+        val androidId: String? = Settings.Secure.getString(contentResolver, Settings.Secure.ANDROID_ID)
+        if (androidId?.isEmpty() == false) {
```

Comment:
@tuancoltech The `ANDROID_ID` should never be empty, so I don't this code is needed?

https://developer.android.com/reference/android/provider/Settings.Secure#ANDROID_ID